test/codegen/javascriptparser.js

```diff
@@ -14,7 +14,7 @@
 
 'use strict';
 
-const JavascriptParser = require('./../../lib/codegen/javascriptparser');
+const JavascriptParser = require('./../../scripts/codegen/javascriptparser');
```

Comment:
I think this is used at runtime by Cicero to introspect the generated JS code and to grab annotations on it.

Reply:
Good catch. It's used right here (inherited from the ScriptManager):
https://github.com/accordproject/ergo/blob/4597c52f59f2824ced0a27410d66634a42065cfe/packages/ergo-compiler/lib/script.js#L18

I am unclear why this is part of Concerto though. I couldn't find a specific used for it. I would be ok moving that file to the Ergo repository (eventually this could be entirely removed, since we now always use JS files generated by the Ergo compiler).

Reply:
> Good catch. It's used right here (inherited from the ScriptManager):
> https://github.com/accordproject/ergo/blob/4597c52f59f2824ced0a27410d66634a42065cfe/packages/ergo-compiler/lib/script.js#L18
> 
> I am unclear why this is part of Concerto though. I couldn't find a specific used for it. I would be ok moving that file to the Ergo repository (eventually this could be entirely removed, since we now always use JS files generated by the Ergo compiler).

I take it back, it's used at least in the scripts to generate the API...

Reply:
Hi Jerome. I'd love to see this code removed from Concerto as it drags in a lot of dependencies (acorn etc). We should create a new module for it so we can use it as a `devDependency` for the build system.